pyorientAssignment.py

```diff
@@ -36,14 +36,41 @@
 # then test each price to see if it is smaller than the current minimum. If it is, update 
 # the minimum variable with that price. You can do something similar to find the maximum.
 
+price = []
 for record in records:
-	print record.price
+   price.append(record.price)
+   
+totalPrice = 0
+
+for i in price:
+    totalPrice += i
+
+averagePrice = totalPrice / len(records)
+print 'average price: ' + str(averagePrice)
+
+highPrice = 500000000000000000
+
+for i in price:
+    if i > highPrice:
+        i = highPrice
+    else:
+        highPrice = highPrice
+
+print 'min price: ' + str(i)
```

Comment:
In this case you are calculating the max price. You should use your highPrice variable, though, which is storing this value. In this case 'i' is only storing the last price in the loop.